search/processor.cpp

```diff
@@ -451,6 +453,30 @@ void Processor::SearchCoordinates()
   m_emitter.Emit();
 }
 
+void Processor::SearchPlusCode()
+{
+  string code;
+  if (openlocationcode::IsFull(m_query))
+  {
+    code = m_query;
+  }
+  else if (openlocationcode::IsShort(m_query))
+  {
+    if (GetPosition().IsAlmostZero())
+      return;
+    ms::LatLon const & latLon = MercatorBounds::ToLatLon(GetPosition());
+    code = openlocationcode::RecoverNearest(m_query, {latLon.lat, latLon.lon});
+  }
+  if (!code.empty())
+  {
+    openlocationcode::CodeArea result = openlocationcode::Decode(code);
+    m_emitter.AddResultNoChecks(
+        m_ranker.MakeResult(RankerResult(result.GetCenter().latitude, result.GetCenter().longitude),
+                            true /* needAddress */, true /* needHighlighting */));
```

Comment:
Why do you need highlighting here?

Reply:
I have no idea what this affects, copied from `SearchCoordinates()`. @tatiana-yan?

Reply:
я вообще не в курсе что делает needHighlighting
надо разобраться или вопрос был риторический?

Reply:
The result of SearchCoordinates() (and of SearchPlusCode, I think) is a pair of coordinates which we highlight because it matches what the user typed. I'm not even sure we have to highlight it there. But here you don't want to highlight a digit in the coordinate because it matches a digit in the plus code, do you?

Reply:
You are right, I've changed it to `false`.